README.md

```diff
@@ -1,11 +1,12 @@
 [openFrameworks](http://openframeworks.cc/)
 ================
 
-openFrameworks is a C++ toolkit for creative coding.  If you are new to OF, welcome!
```

Comment:
Although the proposed change is more terse, it also changes the tone of voice. some might prefer a more conversational tone of voice.